app/assets/javascripts/darkswarm/services/stripe_elements.js.coffee

```diff
@@ -35,11 +40,17 @@ Darkswarm.factory 'StripeElements', ($rootScope, Messages) ->
           secrets.cc_type = @mapPaymentMethodsApiCardBrand(response.paymentMethod.card.brand)
           secrets.card = response.paymentMethod.card
           submit()
+      .catch (response) =>
+        # Stripe handles errors in the response above. This code may never be
+        # reached. But if we get here, we want to know.
+        @reportError(response, t("js.stripe_elements.unknown_error_from_stripe"))
+        throw response
```

Comment:
I know this is duplication but eventually this Stripe SCA part will replace the old Stripe code and we will have only one version.

Reply:
yes, I kept it duplicated also because stripeconnect will go and it will be easier to remove if separated.